### PR TITLE
Update wording and steps

### PR DESCRIPTION
**Pull request by gh-andrem**
- Change wording as toast sounds misleading
- Add steps on how to search for URL in M365 Defender portal

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/security/defender-endpoint/web-content-filtering.md [hunk 1]
```diff
@@ -112,7 +112,7 @@ A panel will open where you can select the priority and add additional details s
 
 ### URL category lookup
 
-To determine the category of a website, you can use the URL search function available on the Microsoft 365 Defender portal (https://security.microsoft.com). In the URL search results, the web content filtering category appears under **URL/Domain details**. Administrators can also dispute the category of the domain directly from this page, as shown in the image below. If the category result is not shown, the URL is not currently assigned to an existing web content filtering category.
+To determine the category of a website, you can use the URL search function available on the Microsoft 365 Defender portal (https://security.microsoft.com) under **Endpoints** > **Search**. In the URL search results, the web content filtering category appears under **URL/Domain details**. Administrators can also dispute the category of the domain directly from this page, as shown in the image below. If the category result is not shown, the URL is not currently assigned to an existing web content filtering category.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/web-content-filtering.md]:
```suggestion
To determine the category of a website, you can use the URL search function available on the Microsoft 365 Defender portal (https://security.microsoft.com) under **Endpoints** > **Search**. In the URL search results, the web content filtering category appears under **URL/Domain details**. Administrators can also dispute the category of the domain directly from this page, as shown in the following image. If the category result is not shown, the URL is not currently assigned to an existing web content filtering category.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @amueller-tf, kindly also commit the additional suggested change. Many thanks.

**Comment by gh-andrem** [comment]
Thanks @JohanFreelancer9 Made the change.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by denisebmsft — APPROVED** [review]
Thank you @amueller-tf @yogkumgit @JohanFreelancer9

**Pull request merged**